### PR TITLE
Enh/meclx3519

### PR DESCRIPTION
**Pull request by silkenelson**
This pull request include:
- support for Alvium
- support for new time tool (win use in MEC)
- EPICS & BLD text file extraction as used in MEC

### CODE REVIEW

**Review by vespos — APPROVED** [review]
Just a couple comments.
The submit_plots will only work if the --bld or --pedestal argument are passed at the moment, is that intentional?

File: smalldata_tools/DetObject.py [hunk 1]
```diff
@@ -142,6 +143,7 @@ def DetObject(srcName, env, run, **kwargs):
        33: Epix10k2MObject, #this is a quad-lookds like a 2M
        34: CameraObject,
        36: iStarObject,
+       37: AlviumObject,
```
[the remark this answers is not in the document]
Comment (silkenelson) [reply smalldata_tools/DetObject.py:146]: What do you mean with duplicates? The psana.Detector as a .dettype, that is the number I'm using to determine which type of object to use. So if I'm using the same object for different numbers, that is ok, but the opposite would be a mistake.

File: arp_scripts/submit_plots.sh [hunk 2]
```diff
@@ -69,10 +75,12 @@ export MYDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
 source /reg/g/psdm/etc/psconda.sh -py3
 #conda activate ana-4.0.16-py3
-ABS_PATH=`echo $MYDIR | sed  s/arp_scripts/legacy/g`
-PLOT_PY=DataqualityPlots
+ABS_PATH=`echo $MYDIR | sed  s/arp_scripts/producers/g`
+PLOT_PY=FirstEventPlots
```
[the remark this answers is not in the document]
Comment (silkenelson) [reply arp_scripts/submit_plots.sh:79]: Oh, that should go, I mostly wanted to add the BldEpics option. 'FirstEventPlots' was a rather meclx3519 specific plot. I did not want to remove DataQuality plots, but for this experiment  we run 'FirstEventPlots' instead. I'm not super thrilled with keeping the plotting files with the producers, but the produce something (mostly plots, or text-files like the BldEpics code).

[the remark this answers is not in the document]
Comment (vespos) [reply arp_scripts/submit_plots.sh:79]: Sounds good! We can come back to this when we actually implement more robust / permanent plots here

**Pull request merged**